crafters/image/ImageNormalizer/README.md

```diff
@@ -1,3 +1,48 @@
 # ImageNormalizer
 
-it receives values of file names on the doc-level and returns image matrix on the chunk-level 
\ No newline at end of file
+**ImageNormalizer** is a crafter that normalizes an input image.
+
+The **ImageNormalizer** executor needs the following parameters:
+
+| `param_name`  | `param_remarks` |
+| ------------- | ------------- |
+| `target_size`  |It's the desired output size.  |
+| `img_mean`  |It's the mean of the images in `RGB` channels.  |
+| `img_std`  |It's the std of the images in `RGB` channels.  |
+| `resize_dim`  |It's the desired size that images will be resized to. They are resized before the cropping |
```

Comment:
there is no cropping in this executor involved right?

Reply:
I think it is, around here:
`img, _, _ = _crop_image(img, target_size=self.target_size, how='center')`

Reply:
Ok my bad